modules/planning/common/open_space_info.h

```diff
@@ -229,6 +229,7 @@ class OpenSpaceInfo {
   }
 
   const std::vector<TrajGearPair> &paritioned_trajectories() const {
+    // to-do: export to chart
```

Comment:
TODO(Who): xxx.

Reply:
> TODO(Who): xxx.

fixed

Reply:
Not yet. Should exactly start with `TODO(Renxin): `.